sycl/source/detail/device_impl.cpp

```diff
@@ -296,6 +296,10 @@ bool device_impl::has(aspect Aspect) const {
     return is_accelerator();
   case aspect::custom:
     return false;
+  case aspect::emulated:
+    return false;
```

Comment:
Could you please add a TODO comment here reminding us to implement this for the FPGA emulator and ESIMD emulator in the future?